### ISSUE TITLE
The used time to run this code?

### ISSUE THREAD

**Issue by jymhero** [InnovArul/rgb_IR_personreid#5]
I run this code in GeForce GTX 1070, and each epoch costs 8 mins.
20000 epochs may cost 200 days.....
I not sure what's wrong.

**Comment by InnovArul** [comment]
Hi,

I think you need not run 20000 epochs. I forgot the exact number of epochs I used to run. But its not more than 100 (maybe 60?). You can see when the training loss flattens and stop the training.

You are right. 20000 is misleading. But the reason is that, when I developed the code, I wanted to run the training indefinitely and test each epoch. 

Hope it helps.
